src/Components/Home.js

```diff
@@ -1,46 +1,36 @@
 import React, { Component } from 'react';
 import { Link } from 'react-router-dom';
-import './Home.scss';
+import '../scss/pages/Home.scss';
 import logo from '../logo.svg';
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
 
 export default class Home extends Component {
   render() {
     return (
-      <div className="Home">
-        <header className="App-header">
-          <img src={logo} className="App-logo" alt="logo" />
-          <p>
-            <code>Initiative</code>
-          </p>
-        </header>
-        <section>
-          <Link to="/CharacterCreate">
-            <div>
-              <FontAwesomeIcon icon="user-plus" />
-              <p>Create New Character</p>
-            </div>
+      <React.Fragment>
+        <div className="app-logo">
+          <img src={logo} className="app-logo__icon" alt="logo" />
+          <div className="app-logo__name">Initiative</div>
+        </div>
+        <section class="navigation list">
+          <Link to="/CharacterCreate" class="list__item list__item--icon">
```

Comment:
This should be `className`. There are a few other spots in this file that you use `class`, let's change these to `className`.